### PR TITLE
Implement God Mode

### PR DESCRIPTION
**Pull request by Lesmiscore**
## Please follow the guide below

- You will be asked some questions, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *pull request* (like that [x])
- Use *Preview* tab to see how your *pull request* will actually look like

---

### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [contributing guidelines](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#developer-instructions) including [yt-dlp coding conventions](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#yt-dlp-coding-conventions)
- [x] [Searched](https://github.com/yt-dlp/yt-dlp/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests
- [x] Checked the code with [flake8](https://pypi.python.org/pypi/flake8)

### In order to be accepted and merged into yt-dlp each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [ ] Bug fix
- [x] Improvement
- [ ] New extractor
- [x] New feature

---

### Description of your *pull request* and other information

This PR implements "God Mode" for anyone interested in. Using it will make you the God, but since we're just humans there's nothing we can help.

TLDR: Ignore `--allow-unplayable-formats` when the user isn't running from source.

### CODE REVIEW

**Comment by Ashish0804** [comment]
While I do find this funny and can see why some people may think we need this...
I am against this.
I myself use that option for testing (with binary)
But I now use my own compiled binary so I will be very happy if u can add a `--ignore-god-mode-check` option that can be used when compiling and then I have nothing against this being merged :D

**Comment by fstirlitz** [comment]
Might be more useful to modify `bug_reports_message` if `--allow-unplayable-formats` is passed.

**Comment by pukkandan** [comment]
The amount of issues opened with DRM is not much nowadays and is only a tiny portion of the total amount of invalid/incomplete issues we receive. There is already a warning shown when `--allow-unplayable` is used. I don't see much point in modifying the `bug_reports_message`

```
WARNING: You have asked for UNPLAYABLE formats to be listed/downloaded. This is a developer option intended for debugging.
         If you experience any issues while using this option, DO NOT open a bug report
```
